projects/npm-tools/packages/npm-scripts/src/typescript/getTypeScriptDependencyGraph.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const fs = require('fs');
+const path = require('path');
+
+const getWorkspaces = require('../utils/getWorkspaces');
+
+/**
+ * Across the set of Yarn workspaces, determine the dependency graph of
+ * TypeScript projects.
+ *
+ * Note that in the future, `tsc` itself may be able to do this for us.
+ *
+ * See: https://github.com/microsoft/TypeScript/issues/25376
```

Comment:
Loving the energy in the last comment:

![Screen_Shot_2021-03-31_at_12_49_06](https://user-images.githubusercontent.com/905006/113139904-9a948c00-921f-11eb-83b3-627559a0d480.png)

![shutupandtakemymoney](https://user-images.githubusercontent.com/905006/113139920-a1230380-921f-11eb-818e-8a63ed393f62.gif)

Reply:
Yeah, just thinking how wonderful it would be if we could wrap `ant`/`gradle` in Bazel...